Jenkinsfile

```diff
@@ -1,16 +1,32 @@
 #!/usr/bin/env groovy
 
-dockerfile {
-    slackChannel = '#ksqldb-quality-oncall'
-    upstreamProjects = 'confluentinc/schema-registry'
-    extraDeployArgs = '-Ddocker.skip=true'
-    dockerPush = false
-    dockerScan = false
-    dockerImageClean = false
-    downStreamRepos = ["confluent-security-plugins", "confluent-cloud-plugins"]
-    nanoVersion = true
-    maxBuildsToKeep = 99
-    maxDaysToKeep = 90
-    extraBuildArgs = "-Dmaven.gitcommitid.nativegit=true"
+parallel unitTests: { // Skip the integration tests.  The unit tests take longer the the ITs, so we push the spotbugs and Maven site goals to the other job.
```

Comment:
typo: longer than. After this change, the integration tests will take longer than the unit tests right, so should we revise the comment?